lib/dataloader.ex

```diff
@@ -52,13 +89,22 @@ defmodule Dataloader do
           options: [option]
         }
 
-  @type option :: {:timeout, pos_integer}
+  @type option :: {:timeout, pos_integer} | {:get_policy, atom()}
   @type source_name :: any
 
   @default_timeout 15_000
+  def default_timeout, do: @default_timeout
+
+  @default_get_policy Application.get_env(:dataloader, :get_policy, :raise_on_error)
```

Comment:
Ah so this pattern can lead to some fun behaviour for users. They'll do something like:

```
config :dataloader, get_policy: :return_nil_on_error
```

and then build their project from scratch. Then they change their mind and remove it. _the ` @default_get_policy` value will stay the same_. The value was pulled when the dependency was compiled, and the dependency will not be recompiled just because the config changed.

If we're going to accept a mix config option here we need to always retrieve it at runtime.